qt-bugs@ issue : N207415
Trolltech task ID : 207597
bugs.kde.org number : none
applied: no
author: Dirk Mueller <mueller@kde.org>

This patch fixes random crashes due to paintevent
recursion and in once case a 100% startup crash (lyx).

--- a/src/gui/widgets/qstatusbar.cpp
+++ b/src/gui/widgets/qstatusbar.cpp
@@ -643,7 +643,7 @@
     }
 
     emit messageChanged(d->tempItem);
-    repaint();
+    update();
 }
 
 /*!
